pages/common/test.md

```diff
@@ -20,6 +20,6 @@
 
 `test ! -d "{{path/to/directory}}"`
 
-- If A is true, then do B, or C in the case of an error (notice that C may run even if A fails):
+- If condition is true, echo "true" runs, if the condition is false then echo "false" runs:
```

Comment:
I'm not sure this is a big improvement. The example itself is quite self-explainatory, so I think we can keep the description more general.

Reply:
Can this conversation be marked as resolved?